### PR TITLE
Document Qt >= 5.6 not working in Linux

### PR DESCRIPTION
**Pull request by snappizz**
i feel rather silly filing separate PRs for changes to a single file, but they're orthogonal problems and smaller changes get merged faster

### CODE REVIEW

File: README_LINUX.md [hunk 1]
```diff
@@ -45,6 +45,7 @@ Build requirements (optional features)
    * http://qt-project.org
    * cross-platform graphical user interface library, for Qt IDE and sclang's Qt GUI kit
    * Qt >= 5.0 should work, but build-test is done against 5.3
+   * As of this writing, Qt >= 5.6 will NOT work.
```
Comment (timblechmann) [inline README_LINUX.md]: compile qt with qtwebkit from sources.

Comment (snappizz) [inline README_LINUX.md]: [qtwebkit appears to be a dead project.](https://trac.webkit.org/wiki/BuildingQtOnLinux) but i'll try anyway...

Comment (snappizz) [inline README_LINUX.md]: Continuing at #2204

**Comment by patrickdupuis** [comment]
If there are known issues with using Qt >= 5.6, then we need to given an idea of what those are. I have SC 3.7.2 built with Qt 5.6.1 running on Fedora 24. What should I be worried

**Comment by patrickdupuis** [comment]
about?

Oops...

**Comment by patrickdupuis** [comment]
If the issue is the missing QtWebkit, then it is not a linux issue, it's a distro specific issue. The fedora 24 repositories, for example, offer Qt 5.6.1 and QtWekbit 2.3.4.

**Comment by snappizz** [comment]
@patrickdupuis  See #2204

**Comment by patrickdupuis** [comment]
I'm aware of #2204. There's two of us who have reported  building SC with Qt5.6.1 no problem. We should not be leaving notes in the readme that give the impression you can't build SC with Qt>=5.6. The real issue is something else.

**Comment by snappizz** [comment]
The wording I introduced is intentionally very soft: "As of this writing, there are known issues with building on Qt >= 5.6." Some people are seeing issues, some aren't. I wanted to put up a quick notice in the README to save some frustration for Linux users building from master.

**Comment by patrickdupuis** [comment]
How about saying, "some users have reported issues building SC with Qt >=5.6 on some linux distros". That way we mention the issue is when building (not that it will crash all the time).

**Comment by crucialfelix** [comment]
I actually misread it the first time as "there are NO known issues"

I agree there could be documentation about the known issues. But we should use the github wiki for that kind of thing so it can be updated often and easily and just link to that in the documentation.

(Also we should switch to using the github wiki instead of doing it all in jekyll. Jekyll should just be the website and move the pages out of it. )

**Comment by gusano** [comment]
> Also we should switch to using the github wiki instead of doing it all in jekyll. Jekyll should just be the website and move the pages out of it

oh yes :+1:

**Pull request merged**